graphite-ch-optimizer.go

```diff
@@ -165,11 +164,11 @@ func processFlags() error {
 	// ClickHouse set
 	fc := pflag.NewFlagSet("clickhouse", 0)
 	fc.StringP("server-dsn", "s", viper.GetString("clickhouse.server-dsn"), "DSN to connect to ClickHouse server")
-	fc.Duration("optimize-interval", viper.GetDuration("clickhouse.optimize-interval"), "The active partitions won't be optimized more than once per this interval, seconds")
+	fc.Duration("optimize-interval", viper.GetDuration("clickhouse.optimize-interval"), "The partition will be merged unless it have active writes for more than, duration")
```

Comment:
`The partition will be merged after having no writes for more than the given duration`

Reply:
Many thanks